src/components/new-member/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import Image from "next/image";
+
+const NewMembersCard = ({
+  newMemberName,
+  newMemberImageSrc,
+}: {
+  newMemberName: string;
+  newMemberImageSrc: string;
+}) => {
+  return (
+    <div className="flex bg-white max-w-sm rounded-md transition-shadow duration-300 h-35 overflow-hidden">
+      <div className="flex-col p-4 cursor-default flex-1 rounded-md border-0 shadow-none">
```

Comment:
- Remove default css properties: cursor-default, border-0 and shadow-none.
- Check if image and heading needs to be centerd.

Reply:
removed